pramen-py/src/pramen_py/app/__init__.py

```diff
@@ -18,6 +18,6 @@
 
 
 env = Env(expand_vars=True)
-env.read_env()
+env.read_env(recurse=False)
```

Comment:
Does this allow loading the library if `.env` is not present in the current directory?

Reply:
```python
try
  env.read_env()
catch
  OsError: env.read_env(recurse=False)
```